merp_reservation_strategy/views/res_config.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_stock_config_settings" model="ir.ui.view">
+        <field name="name">Ventor/mERP Picking Wave - Stock Settings</field>
```

Comment:
Should be just "Picking Wave - Stock Settings". Because it is not even related to Ventor App or mERP. It is generic setting.
Also it is not related to Picking Wave, but to all pickings I guess? Maybe even better name is needed?